src/Calculator.java

```diff
@@ -0,0 +1,28 @@
+public class Calculator {
+
+    public void rectangleArea(Rectangle rectangle) {
```

Comment:
What does this method do? Naming! Method names should be verbs and tell us what method does. Yours doesn't tell us what it does.